.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
You don't need to change it on this PR, but it's better to put these extra changes in a separate PR instead of lumping them in. Count-intuitively it makes it faster to separate stuff because when the PR only has a single purpose it's easier to just figure out if it's accomplishing that one thing. Like if this editorconfig were it's own PR, it would take less than a minute to review because it's so simple.

Reply:
Oh, they're not supposed to be different. I'm supposed to be copy/pasting the same stuff, but there's a diff for some reason :shrug:

Let me try copy/pasting better.